WordPress/Classes/ViewRelated/Portfolio/ProjectTableViewCell.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+
+class ProjectTableViewCell: BasePageListCell {
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var menuButton: UIButton!
+    @IBOutlet var featuredImageView: UIImageView!
+    @IBOutlet var featuredImageWidthConstraint: NSLayoutConstraint!
+    @IBOutlet var featuredImageHeightConstraint: NSLayoutConstraint!
+
+    override var post: AbstractPost? {
+        didSet {
+            configureTitle()
+            configureImage()
+        }
+    }
+
+    // MARK: - Life Cycle
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        applyStyles()
+
+        featuredImageView.clipsToBounds = true
+        featuredImageView.contentMode = .scaleAspectFill
+    }
+
+    // MARK: - Configuration
+
+    func applyStyles() {
+        WPStyleGuide.applyPageTitleStyle(titleLabel)
+    }
+
+    func configureTitle() {
+        guard let post = self.post else {
+            return
+        }
+
+        let rightPost = getRightPost(from: post)
+        let str = rightPost.titleForDisplay() ?? ""
+        titleLabel.attributedText = NSAttributedString(string: str, attributes: WPStyleGuide.pageCellTitleAttributes() as? [NSAttributedStringKey: Any])
+    }
+
+    func configureImage() {
+        guard let post = self.post else {
+            return
+        }
+
+        let rightPost = getRightPost(from: post)
+
+        if let featuredImagePath = rightPost.pathForDisplayImage, let featuredImageURL = URL(string: featuredImagePath) {
```

Comment:
By replacing this statement with a `guard let else`, you can move up here the `featuredImageView.image = nil` (along with a return).

That way you can reduce the indentation level in this whole snippet (which makes things super readable!)